vcpkg/consume/asset-caching.md

```diff
@@ -28,22 +27,25 @@ In this tutorial you'll learn how to:
 
 ## 1 - Create an asset cache directory
 
-You can use the Azure Blob Storage backend "[`x-azurl`](../users/assetcaching.md#x-azurl)", to store
-assets. Alternatively, you can use filesystem directories to store
-your download assets, including network locations. To do so, you need to use the
-`file://` protocol in your asset cache URL.
+vcpkg offers two types of asset cache providers:
```

Comment:
```suggestion
vcpkg offers the following asset cache providers:
```

This way we don't have to edit that line (or, more likely, forget to edit that line) if we ever add more.